src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -264,30 +262,25 @@ private void processMap(Document doc, Element mapElement, Map<?, ?> map) {
 
             if (key == null) {
                 addNode(doc, mapItemElement, "key");
-            }
-            else if (key instanceof Struct) {
+            } else if (key instanceof Struct) {
                 final Element mapItemKeyElement = addNode(doc, mapItemElement, "key");
                 processStruct(doc, mapItemKeyElement, (Struct) key);
-            }
-            else if (key instanceof String) {
+            } else if (key instanceof String) {
                 final Element mapItemKeyElement = addNode(doc, mapItemElement, "key");
                 mapItemKeyElement.setTextContent(key.toString());
-            }
-            else if (key instanceof byte[]) {
+            } else if (key instanceof byte[]) {
                 final Element mapItemKeyElement = addNode(doc, mapItemElement, "key");
-                mapItemKeyElement.setTextContent(Base64.getEncoder().encodeToString((byte[])key));
+                mapItemKeyElement.setTextContent(Base64.getEncoder().encodeToString((byte[]) key));
             }
             // else if (key.getClass().isArray()) {
             //     addListElements(doc, mapItemElement, ListUtils.nullSafeArrayToList((Object[])key), "key");
             //}
             else if (key instanceof Collection) {
-                addListElements(doc, mapItemElement, (Collection<?>)key, "key");
-            }
-            else if (key instanceof Map) {
+                addListElements(doc, mapItemElement, (Collection<?>) key, "key");
+            } else if (key instanceof Map) {
                 final Element mapItemKeyElement = addNode(doc, mapItemElement, "key");
-                processMap(doc, mapItemKeyElement, (Map<?,?>) key);
-            }
-            else {
+                processMap(doc, mapItemKeyElement, (Map<?, ?>) key);
+            } else {
```

Comment:
```suggestion
            } 
            else {
```

Unrelated to this specific PR